xml/System/DateOnly.xml

```diff
@@ -189,9 +189,9 @@
         <Parameter Name="value" Type="System.DateOnly" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The object to compare to the current instance.</param>
+        <summary>Compares the value of this instance to a specified DateOnly value and returns an integer that indicates whether this instance is earlier than, the same as, or later than the specified DateTime value.</summary>
+        <returns>Less than zero if this instance is earlier than value. Greater than zero if this instance is later than value. Zero if this instance is the same as value.</returns>
```

Comment:
@tarekgh Does the value of the less than or greater than number mean something? For example, if it returns -55, what does it mean? Or do we only expect to get -1, 0 and +1? I ask in case we want to describe that returned value a bit more.

Reply:
We should just mention negative or positive value and not specifying what the actual number will be returned. 

So, the current description looks good to me.